librz/include/rz_flag.h

```diff
@@ -100,6 +100,7 @@ RZ_API RzFlagItem *rz_flag_get(RzFlag *f, const char *name);
 RZ_API RzFlagItem *rz_flag_get_i(RzFlag *f, ut64 off);
 RZ_API RzFlagItem *rz_flag_get_by_spaces(RzFlag *f, ut64 off, ...);
 RZ_API RzFlagItem *rz_flag_get_at(RzFlag *f, ut64 off, bool closest);
+RZ_API RzFlagsAtOffset *rz_flag_get_nearest_list(RzFlag *f, ut64 off, int dir);
```

Comment:
I think this function should not be made an RZ_API... This `RzFlagsAtOffset` is just an internal detail.

Reply:
Use one of the available functions or create a new one that makes sense from the API point of view, but IMO we should avoid exporting this.